samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1430,6 +1430,108 @@
           "Resource": "*"
         }]
       }
+    },
+    "DecodeSSMParameterPolicy": {
```

Comment:
Let's call this `SSMParameterReadPolicy` to match naming convention of other policy templates.